### ISSUE TITLE
Mix binary release 1.0.0 hangs on windows7

### ISSUE THREAD

**Issue by winsvega** [ethereum/mix#222]
Just installed mix 1.0.0 binary release on win7 and it hangs for unlimited time when loading or creating project.

**Comment by DavidBurela** [comment]
I have the same issue. Windows 10 x64.

I can run mix, the UI launches. If I create or open a project, the project loads and then the UI hangs.

**Comment by yann300** [comment]
I can reproduce here (win7). is this happening when resizing the window for you?

**Comment by DavidBurela** [comment]
Yes, I think it may be because I resized the window. After creating the 1st project.
But it crashes on every subsequent load, because the window size is now not the "original" size.

**Comment by LianaHus** [comment]
can't reproduce here as well win 7 pro x64

**Comment by michaltrzesimiech** [comment]
Reproduced with version 1.0.1. on Win7 x64 6.1.7601 SP 1 Build 7601 (res 1366 x 768, which is too low): Mix hangs at starting new project.

Problem signature:
  Problem Event Name:	APPCRASH
  Application Name:	Mix.exe
  Application Version:	0.0.0.0
  Application Timestamp:	56e30b54
  Fault Module Name:	Mix.exe
  Fault Module Version:	0.0.0.0
  Fault Module Timestamp:	56e30b54
  Exception Code:	c0000005
  Exception Offset:	000000000011597e
  OS Version:	6.1.7601.2.1.0.256.1
  Locale ID:	1045
  Additional Information 1:	f4a7
  Additional Information 2:	f4a7fb47bf189f727c56fd43f2a90f40
  Additional Information 3:	16e6
  Additional Information 4:	16e6ce66d9e2f30746c1cba35923a105

**Comment by bobsummerwill** [comment]
@michaltrzesimiech Please can you update to 1.2.2 (https://github.com/ethereum/webthree-umbrella/releases/tag/v1.2.2) and see if you still get the issue?  Thanks!

**Comment by DavidBurela** [comment]
I uninstalled the old version. Installed the new version.
Opened Mix.exe
The main screen loads up with "create new project / open project" and I can click around. 
I create a new project.
The project window opens and project starts loading. I resized the window and it locks up the same

**Comment by DavidBurela** [comment]
Just in case it isn't directly the UI but is something else, here is the output of the console window.

```
Qt: Untested Windows version 10.0 detected!

WARNING: This project is using the experimental QML API extensions for QtWebEngine and is therefore tied to a specific QtWebEngine release.
WARNING: The experimental API will change from version to version, or even be removed. You have been warned!

QtWebEngineProcess not found at location C:/Program Files/Ethereum/bin\QtWebEngineProcess.exe. Trying fallback path...
Layout must be attached to Item elements
Layout must be attached to Item elements
qml: [{"id":0,"jsonrpc":"2.0","method":"eth_accounts","params":[]}]
qml: [{"id":1,"jsonrpc":"2.0","method":"eth_gasPrice","params":[]}]
<Unknown File>: QML QQuickLayoutAttached: Binding loop detected for property "preferredWidth"
<Unknown File>: QML QQuickLayoutAttached: Binding loop detected for property "preferredWidth"
<Unknown File>: QML QQuickLayoutAttached: Binding loop detected for property "preferredWidth"
<Unknown File>: QML QQuickLayoutAttached: Binding loop detected for property "preferredWidth"
<Unknown File>: QML QQuickLayoutAttached: Binding loop detected for property "preferredWidth"
<Unknown File>: QML QQuickLayoutAttached: Binding loop detected for property "preferredWidth"
<Unknown File>: QML QQuickLayoutAttached: Binding loop detected for property "preferredWidth"
qrc:/QtQuick/Controls/ScrollView.qml:232:9: QML Binding: Binding loop detected for property "value"
qrc:/QtQuick/Controls/ScrollView.qml:225:9: QML Binding: Binding loop detected for property "value"
<Unknown File>: QML QQuickLayoutAttached: Binding loop detected for property "preferredWidth"
<Unknown File>: QML QQuickLayoutAttached: Binding loop detected for property "preferredWidth"
<Unknown File>: QML QQuickLayoutAttached: Binding loop detected for property "preferredWidth"
  i  09:17:23.123|main  Killing blockchain & extras database (WithExisting::Kill).
  i  09:17:23.141|main  Killing state database (WithExisting::Kill).
qml: [{"id":1,"jsonrpc":"2.0","method":"eth_gasPrice","params":[]}]
qml: [{"id":0,"jsonrpc":"2.0","method":"eth_accounts","params":[]}]
qml: [{"id":1,"jsonrpc":"2.0","method":"eth_gasPrice","params":[]}]
  i  09:17:23.190|main  Killing blockchain & extras database (WithExisting::Kill).
  i  09:17:23.208|main  Killing state database (WithExisting::Kill).
  i  09:17:23.255|main  Killing blockchain & extras database (WithExisting::Kill).
  i  09:17:23.273|main  Killing state database (WithExisting::Kill).
qml: Closing project
qml: Loading project at file://C:/source/mix/test
qrc:///QtQuick/Controls/Private/ScrollBar.qml:98:46: Unable to assign [undefined] to bool
qrc:///QtQuick/Controls/Private/ScrollBar.qml:98:46: Unable to assign [undefined] to bool
qrc:/QtQuick/Controls/TextArea.qml:728:24: Unable to assign [undefined] to QColor
qrc:///QtQuick/Controls/Styles/Base/TextFieldStyle.qml:75: TypeError: Cannot read property '__contentHeight' of null
qrc:///QtQuick/Controls/Styles/Base/TextFieldStyle.qml:75: TypeError: Cannot read property '__contentHeight' of null
qrc:///QtQuick/Controls/Styles/Base/TextFieldStyle.qml:72:42: Unable to assign TextArea_QMLTYPE_53 to TextField_QMLTYPE_40
qrc:///QtQuick/Controls/Private/ScrollBar.qml:95: TypeError: Cannot read property of null
qrc:///QtQuick/Controls/Private/ScrollBar.qml:95: TypeError: Cannot read property of null
qml: Unable to initiate request to the live network. Please verify your ethereum node is up. Error status: 0
qml:
qml: Unable to initiate request to the live network. Please verify your ethereum node is up. Error status: 0
qml:
qml: Unable to initiate request to the live network. Please verify your ethereum node is up. Error status: 0
qml:
qml: Unable to initiate request to the live network. Please verify your ethereum node is up. Error status: 0
qml:
qml: Unable to initiate request to the live network. Please verify your ethereum node is up. Error status: 0
qml:
qml: Closing project
qml: Creating project mixtest at file://C:/temp
qml: Closing project
qml: Loading project at file://C:/temp/mixtest
  i  09:18:04.938|main  Killing blockchain & extras database (WithExisting::Kill).
  i  09:18:04.954|main  Killing state database (WithExisting::Kill).
Waiting for current execution to complete
  i  09:18:06.049|<unknown>  Decreasing DB node ref count below zero with no DB node. Probably have a corrupt Trie. #84acdce9ÔÇª
  i  09:18:06.066|<unknown>  Decreasing DB node ref count below zero with no DB node. Probably have a corrupt Trie. #a71b164eÔÇª

```

**Comment by bobsummerwill** [comment]
Thanks for digging more, @DavidBurela!

**Comment by adarshsunil** [comment]
Same issue happen in Ubuntu also when opening mix...
<Unknown File>: QML QQuickLayoutAttached: Binding loop detected for property "preferredWidth"
 ℹ  04:08:46 PM.584|mix  Killing blockchain & extras database (WithExisting::Kill).
  ℹ  04:08:47 PM.438|mix  Killing state database (WithExisting::Kill).
mix: symbol lookup error: /usr/lib/libweb3jsonrpc.so: undefined symbol: _ZN7jsonrpc9ProcedureC1ERKSsNS_22parameterDeclaration_tENS_10jsontype_tEz

please help

**Comment by michaltrzesimiech** [comment]
@bobsummerwill I did tried that for 1.2.2. It seems that it contains the same version of Mix, which is 1.0.1.

My original message (deleted by accident):
Reproduced with version 1.0.1. on Win7 x64 6.1.7601 SP 1 Build 7601 (res 1366 x 768, which is too low): Mix hangs at starting new project.

Problem signature:
Problem Event Name: APPCRASH
Application Name: Mix.exe
Application Version: 0.0.0.0
Application Timestamp: 56e30b54
Fault Module Name: Mix.exe
Fault Module Version: 0.0.0.0
Fault Module Timestamp: 56e30b54
Exception Code: c0000005
Exception Offset: 000000000011597e
OS Version: 6.1.7601.2.1.0.256.1
Locale ID: 1045
Additional Information 1: f4a7
Additional Information 2: f4a7fb47bf189f727c56fd43f2a90f40
Additional Information 3: 16e6
Additional Information 4: 16e6ce66d9e2f30746c1cba35923a105
